### ISSUE TITLE
Link to CS@NYU discord

### ISSUE THREAD

**Issue by calvinatian** [BUGS-NYU/nyu-cs-wiki#99]
There should probably be a link to the CS@NYU discord somewhere. Not sure exactly where.

**Comment by jk-fu** [comment]
Do you have a discord invite link that doesn't expire and has no limit?

**Comment by calvinatian** [comment]
@jk-fu yeah you can use this link from their discord: https://discord.gg/2pq5dWt

**Comment by Ai80-8** [comment]
Delete it all.
